charters/Architecture/charter.md

```diff
@@ -0,0 +1,48 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including
+identifying and addressing *cross-cutting* architecture, integration, and operational concerns across DCP
+projects.
+
+The core team is composed of *Technical Leads* representing each technical DCP project group to ensure a diversity of
+skills and views needed for community technical decision-making. 
+
+## In-scope
+
+Software architecture responsibilities
+* Set overall direction of architectural and software design direction for the DCP
+* Provide technical input for the PM product roadmap process
+* Review and provide final approval for technical charters for DCP projects
+* Publish Architecture Requests for Comments (RFCs) documenting technical debt that impacts delivery of DCP as an
+  integrated system
+* Review and provide final approval for cross-project RFC(s)
+    * Available to mentor on all RFC designs
```

Comment:
nit: what is an "RFC design"?   Perhaps "... on all RFCs"?

Reply:
Might also clarify that the charter scope is for **technical* RFCs.